### PR TITLE
Update ddlist.md

### PR DESCRIPTION
**Pull request by ajithpv**
Updated ddlist.md for better clarity.

Signed-off-by: Ajith P Venugopal <ajithpv@outlook.com>

### CODE REVIEW

File: en/object-types/ddlist.md [hunk 1]
```diff
@@ -1,61 +1,61 @@
 ```eval_rst
 :github_url: https://github.com/littlevgl/docs/blob/master/en/object-types/ddlist.md
 ```
-# Drop down list (lv_ddlist)
+# Drop-down list (lv_ddlist)
```
Comment (ajithpv) [inline en/object-types/ddlist.md]: According to [standard name](https://en.wikipedia.org/wiki/Drop-down_list), I have changed the 'Drop down list' to Drop-down list. Could you please confirm this is OK for LittlevGL naming conventions?

Comment (embeddedt) [reply en/object-types/ddlist.md]: It's fine by me; I don't think @kisvegabor will have an issue with it either.

Comment (kisvegabor) [reply en/object-types/ddlist.md]: I'm absolutely fine with it if it's the standard name.

**Comment by ajithpv** [comment]
@embeddedt and @kisvegabor : Could you please merge the PR?

**Comment by kisvegabor** [comment]
Sure!

Thank you!

**Pull request merged**